libs/test/SalesforceSDKTest/src/com/salesforce/androidsdk/accounts/UserAccountTest.java

```diff
@@ -217,6 +347,17 @@ private JSONObject createTestAccountJSON() throws JSONException{
         object.put(UserAccount.LOCALE, TEST_LOCALE);
         object.put(UserAccount.PHOTO_URL, TEST_PHOTO_URL);
         object.put(UserAccount.THUMBNAIL_URL, TEST_THUMBNAIL_URL);
+        object.put(UserAccount.LIGHTNING_DOMAIN, TEST_LIGHTNING_DOMAIN);
```

Comment:
Test should include all the fields in case of a copy / paste error when assigning fields from map values etc.